config/platforms/docJson.ts

```diff
@@ -1,5 +1,5 @@
-import StyleDictionary from 'style-dictionary'
-import {PlatformInitializer} from '~/types/PlatformInitializer'
+import type StyleDictionary from 'style-dictionary'
```

Comment:
It's really to distinguish this import from the usual style dictionary object one.

Reply:
Fine for me, is this something you think we should do in general? Do you feel it is or is not important to be aware if a type comes from style dictionary or from us? I felt this way its clear it is from style dictionary, but maybe this is not important to know.

Reply:
I see, that's a fair point. It's a tricky one, because just a few files down, you're doing an explicit destructured import for React.FC. Arguably, that should also not be a separate import either? I think I agree with your initial approach, but ask that we be consistent with all imports from node_modules then and retain the namespace.

Reply:
`React.FC`? 🤨 I don't think I ever used react. Or is this storybook? I agree we should have a consistent rule. Is this enforceable through eslint?

Reply:
I created https://github.com/primer/primitives/issues/411 I think this should be maybe addressed in a separate PR/Issue and not in this one, right?